inference-engine/src/transformations/include/ngraph_ops/nms_ie.hpp

```diff
@@ -58,5 +58,34 @@ class TRANSFORMATIONS_API NonMaxSuppressionIE2 : public NonMaxSuppressionIE {
     std::shared_ptr<Node> clone_with_new_inputs(const OutputVector & new_args) const override;
 };
 
+class TRANSFORMATIONS_API NonMaxSuppressionIE3 : public Op {
+public:
+    static constexpr NodeTypeInfo type_info{"NonMaxSuppressionIE", 3};
```

Comment:
Please use `NGRAPH_RTTI_DECLARATION` instead of this code.

Reply:
Fixed.